urllib3/response.py

```diff
@@ -183,7 +186,7 @@ def _init_decoder(self):
             if content_encoding in self.CONTENT_DECODERS:
                 self._decoder = _get_decoder(content_encoding)
 
-    def _decode(self, data, decode_content, flush_decoder):
+    def _decompress(self, data, decode_content, flush_decoder):
```

Comment:
>  The usual implication of things dealing with bytes/strings named decode imply we should be receiving unicode back from the method...

I agree that in the general Python context, decode means between bytes/strings, but I think it's fair that in a transport protocol context it can mean converting from some arbitrary encoding like a binary one (e.g. compression) to a native format.